ivy/functional/ivy/general.py

```diff
@@ -1497,6 +1497,17 @@ def cumprod(
     -------
     ret
         Input array with cumulatively multiplied elements along axis.
+        
+    Examples
+    --------
+    >>> x = ([a, b, c])
+    >>> ivy.cumprod(x)
+    [a, a * b, a * b * c]
+
+    >>> x = ([a, b, c])
```

Comment:
Maybe add an example that makes use of the `axis` parameter. Include a multi-dimensional array in the same.

Reply:
I added the example, same array but with two axes selections. I chose a simple matrix, in order to be easily understood.